packages/elements-core/src/components/TryIt/Servers/ServersDropdown.tsx

```diff
@@ -0,0 +1,50 @@
+import { faServer } from '@fortawesome/free-solid-svg-icons';
+import { FieldButton, Menu, MenuItem } from '@stoplight/mosaic';
+import { IServer } from '@stoplight/types';
+import { useAtom } from 'jotai';
+import * as React from 'react';
+
+import { chosenServerAtom } from '../state';
+
+export type ServersDropdownProps = {
+  servers: IServer[];
+};
+
+export const ServersDropdown = ({ servers }: ServersDropdownProps) => {
+  const [chosenServer, setChosenServer] = useAtom(chosenServerAtom);
+
+  const serverItems: MenuItem[] = [
+    {
+      type: 'option_group',
+      title: 'Servers',
+      value: chosenServer?.url || '',
+      onChange: url => {
+        const server = servers.find(server => server.url === url);
+        setChosenServer(server);
+      },
+      children: [
+        ...servers.map((server, i) => ({
+          id: server.url,
+          title: server.name || server.description || `Server ${i + 1}`,
```

Comment:
There is a similar code https://github.com/stoplightio/elements/blob/main/packages/elements-core/src/utils/http-spec/IServer.ts#L11

Maybe could be unified somehow, to keep the names consistent in the whole app?

Reply:
Looking at it second time, I find it strange that we use servers from `getServersToDisplay`, which already covers naming issues, but then we do this defensive programming for titles again. 🤔

Reply:
Can prob get rid of the third fallback `Server ${i + 1}` becaus as you pointed out this logic is already handled in IServer, will do. The rest of it though is relevant, because we're basically mapping server props to MenuItem props.